server/src/entities/test.ts

```diff
@@ -0,0 +1,7 @@
+export type TestResults = {
+    patientId: number;
+    testResult: string;
+    typeOfTest: string;
```

Comment:
Same suggestion here for the enum also just more consistent naming to switch the words around and drop the of

```suggestion
    testType: TestType;
```

Reply:
idk if we would want to restrict the test type from our end do we?
if we do what would they be?

Reply:
Yea so we should be controlling the type of data we allow / we accept. The front end is a thin client and shouldn't define anything for the back end. The 2 types of tests we accept are `PCR` and `Antigen` (this is from the Figma mocks and confirmed with the TA)